_data/gaming.yml

```diff
@@ -199,6 +199,14 @@ websites:
       - totp
     doc: https://support.humblebundle.com/hc/en-us/articles/202421374
 
+  - name: Hypixel
+    url: https://hypixel.net/
+    img: hypixel.png
+    tfa:
+      - email
```

Comment:
Could you show where 2FA via email is mentioned?
```suggestion
```

Reply:
![image](https://user-images.githubusercontent.com/43505515/114715847-c8a3c100-9d33-11eb-9998-be7b300758d9.png)
-> Email confirmation